src/Expanders.jl

```diff
@@ -346,15 +350,17 @@ function Selectors.runner(::Type{AutoDocsBlocks}, x, page, doc)
         pages = map(normpath, get(fields, :Pages, []))
         public = get(fields, :Public, true)
         private = get(fields, :Private, true)
+        filterfunc = get(fields, :Filter, x -> true)
         results = []
         for mod in modules
             for (binding, multidoc) in Documenter.DocSystem.getmeta(mod)
                 # Which bindings should be included?
                 isexported = Base.isexported(mod, binding.var)
                 included = (isexported && public) || (!isexported && private)
+                filtered = filterfunc(Core.eval(binding.mod, binding.var))
```

Comment:
`filtered = Base.invokelatest(filterfunc, Core.eval(binding.mod, binding.var))`

Reply:
thanks, that worked like a charm